src/Sulu/Bundle/SecurityBundle/Resources/config/doctrine/Role.orm.xml

```diff
@@ -11,7 +11,11 @@
         <field name="name" type="string" column="name" length="60" unique="true"/>
         <field name="key" type="string" length="60" column="role_key" nullable="true" unique="true"/>
         <field name="system" type="string" column="securitySystem" length="60"/>
-        <field name="anonymous" type="boolean" column="anonymous" nullable="false"/>
+        <field name="anonymous" type="boolean" column="anonymous" nullable="false">
+            <options>
+                <option name="default">false</option>
```

Comment:
Looks like we need to use the following here:
```suggestion
                <option name="default">0</option>
```

Can you test if this works on PGSQL?